tests/functional/services/flows/test_flow_crud.py

```diff
@@ -59,9 +79,26 @@ def test_update_flow(flows_client):
         assert resp[k] == v
 
 
+@pytest.mark.parametrize("value", [None, [], "dummy_value"])
+@pytest.mark.parametrize("key", ["run_managers", "run_monitors"])
+def test_update_flow_request_data(flows_client, key, value):
```

Comment:
I would prefer a more descriptive test name, e.g. `test_update_flow_run_role_serialization`.

That holds for the create flow case as well.

The test name is the primary thing you'll see in the output if it fails. So it can either tell you
- "you broke update flow data"
or
- "you broke serialization of run roles"